core/field.ts

```diff
@@ -797,7 +797,7 @@ export abstract class Field implements IASTNodeLocationSvg,
         scaledWidth += 1 * scale;
         scaledHeight += 1 * scale;
       } else {
-        if (!userAgent.EDGE && !userAgent.IE) {
+        if (!userAgent.EDGE) {
```

Comment:
I looked into this more and concluded that we can [remove the checks for Edge](https://github.com/google/blockly/issues/6339) entirely, and that it makes sense to do so in the same PR since multiple of the checks are for both IE and Edge.

However, I need to load it in Firefox to check that it works after my changes, and I found that Firefox is broken.